server/buildbuddy_server/buildbuddy_server.go

```diff
@@ -117,7 +117,42 @@ func (s *BuildBuddyServer) CreateUser(ctx context.Context, req *uspb.CreateUserR
 }
 
 func (s *BuildBuddyServer) CreateGroup(ctx context.Context, req *grpb.CreateGroupRequest) (*grpb.CreateGroupResponse, error) {
-	return nil, status.UnimplementedError("Not Implemented")
+	auth := s.env.GetAuthenticator()
+	userDB := s.env.GetUserDB()
+	if auth == nil || userDB == nil {
+		return nil, status.UnimplementedError("Not Implemented")
+	}
+	jwtUser := &tables.User{}
+	if err := auth.FillUser(ctx, jwtUser); err != nil {
+		return nil, err
+	}
+
+	groupName := strings.TrimSpace(req.Name)
```

Comment:
generally nicer to use the "Get*" accessors on proto fields so you know you're not getting a nil back.

Can you switch to GetName here and GetAutoPopulateFromOwnedDomain below?

Reply:
Thanks for the tip! Done